cmd/saucectl/saucectl.go

```diff
@@ -42,7 +42,8 @@ func main() {
 
 	commands.AddCommands(cmd, cli)
 	if err := cmd.Execute(); err != nil {
-		panic(err)
+		log.Err(err)
```

Comment:
Don't think that `log.Err()` does anything without `Msg()` ⚠️ 

And instead of two separate calls, you can just do:
`log.Fatal().Err(err).Msg("some reasonable msg here")`

Reply:
You are right. I added a message.